.packit.yaml

```diff
@@ -0,0 +1,46 @@
+# See the documentation for more information:
+# https://packit.dev/docs/configuration/
+
+specfile_path: rubygem-hammer_cli_foreman_leapp.spec
+
+# add or remove files that should be synced
+files_to_sync:
+    - rubygem-hammer_cli_foreman_leapp.spec
+    - .packit.yaml
+
+# name in upstream package repository or registry (e.g. in PyPI)
+upstream_package_name: hammer_cli_foreman_leapp
+# downstream (Fedora) RPM package name
+downstream_package_name: rubygem-hammer_cli_foreman_leapp
+
+upstream_tag_template: "{version}"
+
+actions:
+  post-upstream-clone:
+    - "wget https://raw.githubusercontent.com/theforeman/foreman-packaging/rpm/develop/packages/plugins/rubygem-hammer_cli_foreman_discovery/rubygem-hammer_cli_foreman_discovery.spec -O rubygem-hammer_cli_foreman_discovery.spec"
```

Comment:
discovery -> leapp

Reply:
Thank you so much @ekohl , My bad, I was referencing the Discovery PR. I have corrected it and it seems to build now